src/CMakeLists.txt

```diff
@@ -41,12 +41,16 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 # That prohibits linkage with prebuilt libraries because they aren't compiled with _GLIBCXX_USE_CXX11_ABI=0.
 get_directory_property(OPENVINO_RUNTIME_COMPILE_DEFINITIONS COMPILE_DEFINITIONS)
 include(CMakeDependentOption)
-cmake_dependent_option(BUILD_FAST_TOKENIZERS "Compile core_tokenizers instead of downloading prebuilt library. The option is hidden and forced to ON if _GLIBCXX_USE_CXX11_ABI=0"
-  OFF "NOT _GLIBCXX_USE_CXX11_ABI=0 IN_LIST OPENVINO_RUNTIME_COMPILE_DEFINITIONS" ON)
 if("_GLIBCXX_USE_CXX11_ABI=0" IN_LIST OPENVINO_RUNTIME_COMPILE_DEFINITIONS)
   set(USE_ABI0 ON CACHE BOOL "Set -D_GLIBCXX_USE_CXX11_ABI to 0 for fast_tokenizers")
 endif()
 
+option(BUILD_FAST_TOKENIZERS "Compile core_tokenizers instead of downloading prebuilt library. The option is forced to ON if _GLIBCXX_USE_CXX11_ABI=0" ${USE_ABI0})
+# py_build_cmake doesn't support environment cmake options
+if(DEFINED ENV{BUILD_FAST_TOKENIZERS})
+  set(BUILD_FAST_TOKENIZERS $ENV{BUILD_FAST_TOKENIZERS})
```

Comment:
I suppose it should be cmake option or least let's set value to cache

Reply:
removed